apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -59,13 +60,16 @@ const SongSelector = Radium(class extends React.Component {
   };
 
   changeSong = (event) => {
-    let song = event.target.value;
+    const song = event.target.value;
     this.props.setSong(song);
 
     //Load song
     let options = {id: song};
     options['mp3'] = songs[options.id].url;
     Sounds.getSingleton().register(options);
+
+    //Save song to project
+    project.saveSelectedSong(song);
```

Comment:
There's a chance this immediate save upon changing the selected song interferes with our performance goals, where we're doing some intentional limiting of frequent saves.  I think we should still go ahead and merge this change, but then should think about how to usefully defer this save until the next time it would normally happen.